hazelcast/src/main/java/com/hazelcast/map/impl/operation/MapEntrySetOperation.java

```diff
@@ -39,7 +39,7 @@ public void run() {
         entrySet = recordStore.entrySetData();
         if (mapContainer.getMapConfig().isStatisticsEnabled()) {
             ((MapService) getService()).getMapServiceContext()
-                    .getLocalMapStatsProvider().getLocalMapStatsImpl(name).incrementOtherOperations();
+                    .getLocalMapStatsProvider().getInstantLocalMapStats(name).incrementOtherOperations();
```

Comment:
Do you mean renaming it into a shorter method name, or delegating the method call as getLocalMapStatsProvider(). incrementOtherOperations(); 
?